### PR TITLE
reduce log spam

### PR DESCRIPTION
**Pull request by ZappoMan**
remove some debug messages that are not useful to users... and was just clogging up their log files with useless spam

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: fdd0b48ae13aefbf300d76af1b8065bbc3aaeafb available at: https://gist.github.com/8fd550b531fa54f2e8b1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7450.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7450.dmg

File: libraries/entities/src/EntityTreeElement.cpp [hunk 1]
```diff
@@ -956,8 +956,10 @@ int EntityTreeElement::readElementDataFromBuffer(const unsigned char* data, int
                                 entityItem->recordCreationTime();
                             }
                         } else {
-                            qDebug() << "Recieved packet for previously deleted entity [" <<
+                            #ifdef WANT_DEBUG
+                                qDebug() << "Recieved packet for previously deleted entity [" <<
```
Comment (PhilipRosedale) [inline libraries/entities/src/EntityTreeElement.cpp]: typo: received

**Comment by ZappoMan** [comment]
test plan: smoke test... you shouldn't see these messages in your server logs...

**Comment by hifi-gustavo** [comment]
Build console output for commit: 62a56ecbfa0436f6dd0b1b13fc5dd2d868356581 available at: https://gist.github.com/8fd550b531fa54f2e8b1<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7450.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/HighFidelity-Alpha-PR7450.dmg

**Comment by zzmp** [comment]
@ZappoMan I've seen preprocessor directives both inlined and left-aligned. I couldn't find a standard. Should we add one?

**Comment by ZappoMan** [comment]
@zzmp - I try to indent them like the code around them because I find it easier to read... but you're right, it's a mix in our code. I can go either way.

**Comment by ZappoMan** [comment]
fixes this bug: https://app.asana.com/0/26320649018758/104349881708285

**Comment by PhilipRosedale** [comment]
tested and spam not there for a case when I was often seeing it.

**Pull request merged**